csv_provider/requirements.txt

```diff
@@ -0,0 +1 @@
+kuksa-client==0.3.0
```

Comment:
Maybe better  say "minimum 0.3" (>=), or at least 0.3.x (~=)

Reply:
i opted for 0.3.x